helm/secrets-provider/values.yaml

```diff
@@ -84,3 +84,8 @@ environment:
     # This setting is required.
```

Comment:
If you add an `authnJWT.enabled` setting, then this comment should say:
```
# This setting is required if `authnJWT.enabled` is set to false.
```
Or something like that.